### PR TITLE
feat: CI/CD xyz

### PR DESCRIPTION
**Pull request by csosamorales**
Deploy to xyz [sc-757]

### CODE REVIEW

File: .github/workflows/fleek-deploy.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    paths:
+      - 'ui/**'
+
+jobs:
+  deploy-to-fleek:
+    runs-on: ubuntu-latest
+
+    env:
+      FLEEK_TOKEN: ${{ secrets.FLEEK_TOKEN }}
+      FLEEK_PROJECT_ID: ${{ secrets.FLEEK_PROJECT_ID }}
```
Comment (jsonsivar) [inline .github/workflows/fleek-deploy.yaml:17]: Don't we need more env vars here?

File: .github/workflows/fleek-deploy.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
+name: Deploy site via Fleek
+
+on:
+  pull_request:
+    branches:
+      - main
```
Comment (jsonsivar) [inline .github/workflows/fleek-deploy.yaml:6]: Should we do a separate deployment for main? I'm assuming the site ID in fleek.json is for the dev site.

Comment (csosamorales) [reply .github/workflows/fleek-deploy.yaml:6]: ah, yeah! will separate it

File: ui/src/integrations/ethereum/ethereum.ts [hunk 3]
```diff
@@ -32,6 +32,7 @@ export const Ethereum: Ethereum.Core = {
     return new ethers.Contract(contract.address, contract.abi, provider);
   },
 
+  //TODO remove cause we're using ENS subgraph
```
[the remark this answers is not in the document]
Comment (zoruka) [reply ui/src/integrations/ethereum/ethereum.ts:35]: we were using alchemy to fetch ENS names but we replaced for graphql, maybe is just to not forget to remove

**Pull request merged**